src/shared/config/routes/index.ts

```diff
@@ -0,0 +1,8 @@
+export const enum TypeAppRoute {
+  Welcome = '/',
+  Logout = '/',
```

Comment:
Actually, when you log out you navigate user to the `Welcome` page so no need for `Logout` route

Reply:
This is implemented for clarity and ease of use in the component